docs/welcome.md

```diff
@@ -1,6 +1,57 @@
 ---
 order: 100
 icon: home
+layout: central
 ---
 
-# Welcome
+<style>
```

Comment:
Should we create a global .css file instead of per page, inline css in .md files?

https://retype.com/components/container/#global-css-file